### PR TITLE
Experimental session count

### PR DESCRIPTION
**Pull request by aerosol**
### Changes

Please describe the changes made in the pull request here.

Below you'll find a checklist. For each item on the list, check one option and delete the other.

### Tests
- [ ] Automated tests have been added
- [ ] This PR does not require tests

### Changelog
- [ ] Entry has been added to changelog
- [ ] This PR does not make a user-facing change

### Documentation
- [ ] [Docs](https://github.com/plausible/docs) have been updated
- [ ] This change does not need a documentation update

### Dark mode
- [ ] The UI has been tested both in dark and light mode
- [ ] This PR does not change the UI

### CODE REVIEW

File: lib/plausible/stats/base.ex [hunk 1]
```diff
@@ -132,14 +132,18 @@ defmodule Plausible.Stats.Base do
...
+    q = from(s in "sessions_v2", where: s.site_id == ^site.id)
 
     sessions_q =
-      from(
-        s in "sessions_v2",
-        where: s.site_id == ^site.id,
-        where: s.start >= ^first_datetime and s.start < ^last_datetime
-      )
+      if FunWithFlags.enabled?(:experimental_session_count, for: site) or
+           query.experimental_session_count? do
```
Comment (macobo) [inline lib/plausible/stats/base.ex:142]: Q: do we need this in params/query? If only for tests, could we enable the flag for that specific test and cut down on param-passing?

**Comment by macobo** [comment]
Problem with this flag is we have only a single test for it being on. As such I'd be really careful for turning it on for anything but testing.

Also, related RFC: https://3.basecamp.com/5308029/buckets/35611491/messages/7085680123

File: lib/plausible/stats/base.ex (hunk 1 above)
Comment (aerosol) [reply lib/plausible/stats/base.ex:142]: The intention was for the parameter to be passed down the whole pipeline so we can evaluate the solution for customers who require the fix, before enabling the feature flag

**Comment by aerosol** [comment]
> Problem with this flag is we have only a single test for it being on. As such I'd be really careful for turning it on for anything but testing.

```diff
diff --git a/test/test_helper.exs b/test/test_helper.exs
index 7cbd300df..efed6e079 100644
--- a/test/test_helper.exs
+++ b/test/test_helper.exs
@@ -2,6 +2,7 @@
 Mox.defmock(Plausible.HTTPClient.Mock, for: Plausible.HTTPClient.Interface)
 Application.ensure_all_started(:double)
 FunWithFlags.enable(:window_time_on_page)
+FunWithFlags.enable(:experimental_session_count)
 FunWithFlags.enable(:shields)
 Ecto.Adapters.SQL.Sandbox.mode(Plausible.Repo, :manual)
```

and `mix test` runs fine.

File: lib/plausible/stats/base.ex (hunk 1 above)
Comment (aerosol) [reply lib/plausible/stats/base.ex:142]: Also, feature flags refresh in 15-minute cycles, each node separately.

**Pull request merged**